client/src/catalog-element.html

```diff
@@ -496,6 +496,8 @@ <h1 hidden="[[subElement]]">README.md</h1>
             });
           }
           return el;
+        }).sort(function(x, y) {
+          return x.is < y.is ? -1 : 1;
```

Comment:
Does it matter? It'll just return 1? I guess it might not be a stable sort then?